tests/common/startup/jormungandr_starter.rs

```diff
@@ -0,0 +1,102 @@
+#![cfg(feature = "integration-test")]
+
+extern crate custom_error;
+
+use self::custom_error::custom_error;
+
+use common::configuration::jormungandr_config::JormungandrConfig;
+use common::jcli_wrapper;
+use common::jormungandr_wrapper;
+use common::process_utils;
+use common::process_utils::{
+    output_extensions::ProcessOutput, process_guard::ProcessKillGuard, ProcessError,
+};
+
+use std::process::{Command, Output};
+
+custom_error! {pub StartupError
+    JormungandrNotLaunched{ source: ProcessError } = "could not start jormungandr",
+}
+
+fn try_to_start_jormungandr_node(
+    rest_address: &str,
+    command: &mut Command,
+) -> Result<ProcessKillGuard, StartupError> {
+    println!("Starting jormungandr node...");
+    let process = command
+        .spawn()
+        .expect("failed to execute 'start jormungandr node'");
```

Comment:
This catches something totally unexpected like failure of exec call or missing the executable, right?